templates/auth0-sso-handler-lock10.php

```diff
@@ -13,19 +13,20 @@
   });
 
   var options = <?php echo json_encode( $lock_options->get_sso_options() ); ?>;
+  options.responseType = 'token id_token';
   webAuth.checkSession(options
   , function (err, authResult) {
       if (typeof(authResult) === 'undefined') {
         return;
       }
 
       if (typeof(authResult.code) !== 'undefined') {
-        window.location = '<?php echo add_query_arg( 'auth0', 1, site_url() ); ?>&code=' + authResult.code +
-            '&state=' + authResult.state;
+        window.location = '<?php echo WP_Auth0_Options::Instance()->get_wp_auth0_url(); ?>&code='
+            + authResult.code + '&state=' + authResult.state;
       } else if (typeof(authResult.idToken) !== 'undefined') {
         jQuery(document).ready(function($){
           var $form=$(document.createElement('form')).css({display:'none'}).attr("method","POST").attr("action","<?php
-            echo add_query_arg( 'auth0', 'implicit', site_url() ); ?>");
+            echo add_query_arg( 'auth0', 'implicit', site_url( 'index.php' ) ); ?>");
```

Comment:
what's `index.php`, and why you need to hardcode it here?

Reply:
The main WP entry point. I've tested without and it had issues